assets/js/modules/search-console/util/report-date-range-args.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Report date range args utils.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
```

Comment:
2020 felt like it would never end, but it did 😉 

```suggestion
 * Site Kit by Google, Copyright 2021 Google LLC
```